build.sbt

```diff
@@ -50,13 +51,14 @@ lazy val core = project
     testQuick in Test <<= (testQuick in Test).dependsOn(DynamoDBLocal.Keys.startDynamoDBLocal)
   )
 
-lazy val play24 = project
+lazy val play26 = project
   .dependsOn(core)
   .settings(sharedSettings:_*)
   .settings(
-    name := "configuration-magic-play2.4",
+    name := "configuration-magic-play2.6",
```

Comment:
@alexduf what are the consequence of changing the lib name?

Reply:
Aside from potential confusion, can't think of anything.

Reply:
Would _not_ changing the name be more confusing?

Reply:
yeah it wasn't my smartest move :D
Do rename it though